crates/ra_hir_def/src/nameres.rs

```diff
@@ -137,10 +137,7 @@ impl ModuleOrigin {
     }
 
     pub fn is_inline(&self) -> bool {
-        match self {
-            ModuleOrigin::Inline { .. } => true,
-            ModuleOrigin::CrateRoot { .. } | ModuleOrigin::File { .. } => false,
-        }
+        matches!(self, ModuleOrigin::Inline { .. })
```

Comment:
Note that original pattern is exhaustive, while the new one is not. The first optio is better, as it is more resilient to changes.

Reply:
Done https://github.com/rust-analyzer/rust-analyzer/pull/5105/commits/503f9498cd469c6467250c02bd8f6dbce973dd57